### DISCUSSION TITLE
Welcome to charts Discussions!

### DISCUSSION THREAD

**Discussion by HanSolo** [HanSolo/charts#44] [Announcements]
Aloha everyone, so if you have questions or ideas, please feel free to discuss these things here...

**Comment by tinitus91** [comment]
Hello, 
Surely i missed it but where can i find a simple case to plot a radar char for example ?
Thank you in advance

**Reply by hmf** [reply]
Please start a new thread for new questions. As for the answer, look at the [source code tests](https://github.com/HanSolo/charts/tree/master/src/test/java/eu/hansolo/fx/charts) for examples.

**Reply by HanSolo** [reply]
You might want to check here: [RadarChartTest](https://github.com/HanSolo/charts/blob/master/src/test/java/eu/hansolo/fx/charts/RadarChartTest.java)

**Reply by tinitus91** [reply]
Nice thank you !

**Comment by lilvadim** [comment]
Hello, 
Where can I find some documentation or sample use code?

**Comment by HanSolo** [comment]
I've tried to create a usage demo for each chart in the ([test packages](https://github.com/HanSolo/charts/tree/master/src/test/java/eu/hansolo/fx/charts))

**Reply by samarth-mistry** [reply]
Yes I had seen that but I was getting error in running those, anyways I will try that again, Thank u
